NextcloudTalk/BaseChatViewController.swift

```diff
@@ -1590,52 +1634,82 @@ import QuickLook
         audioFileName += ".mp3"
 
         let activeAccount = NCDatabaseManager.sharedInstance().activeAccount()
+        NCAPIController.sharedInstance().setupNCCommunication(for: activeAccount)
+        let chatFileController = NCChatFileController()
+        chatFileController.initDownloadDirectory(for: activeAccount)
+
+        let destinationFilePath = "\(chatFileController.tempDirectoryPath)/\(audioFileName)"
```

Comment:
Usually it's better to leave appending filesnames/directories to methods like `appendingPathComponent`

Reply:
True, fixed.